packages/rfw/lib/src/dart/text.dart

```diff
@@ -2306,10 +2377,15 @@ class _Parser {
         _advance();
         return _withSourceRange(StateReference(_readParts()), start);
       }
+      if (widgetBuilderScope.contains(identifier)) {
+        final SourceLocation? start = _getSourceLocation();
+        _advance();
+        return _withSourceRange(WidgetBuilderArgReference(identifier, _readParts()), start);
+      }
```

Comment:
let's put this just before the loop identifiers logic (after `set`), to keep the function more consistent.